src/main/java/org/sagebionetworks/bridge/sdk/models/accounts/StudyParticipant.java

```diff
@@ -72,9 +70,7 @@
         this.status = status;
         this.id = id;
         this.languages = languages;
-        if (attributes != null) {
-            this.attributes.putAll(attributes);
-        }
+        this.attributes = attributes;
```

Comment:
I didn't want to put withHealthCode() and a few similar methods on the builder because the SDK user needs to set these (they're read only). It's true that that users could circumvent this by using Jackson, but if hellbent to do so they could send JSON with these values to the server directly... we ignore them.